bindata/v3.11.0/kube-apiserver/pod.yaml

```diff
@@ -9,17 +9,22 @@ metadata:
     revision: "REVISION"
   deletionGracePeriodSeconds: 65 # a bit more than the kube-apiserver shutdown timeout of 60 sec
 spec:
+  initContainers:
```

Comment:
can static pods have init containers?

Reply:
yes, the audit log files are in artifacts